exercises/bob/canonical-data.json

```diff
@@ -127,6 +176,15 @@
       },
       "expected": "Whoa, chill out!"
     },
+    {
+      "uuid": "2a6431bd-43e3-495d-9fff-8834a387329a",
+      "description": "Bob in Vienna",
+      "property": "response",
+      "input": {
+        "heyBob": "OIDA!"
```

Comment:
Let's stick to English for non-unicode examples.

Reply:
Why? Can you tell me an equivalent phrase/word in English then?

Reply:
> Why?

Because Exercism's exercises are written in English. Once we cross the mutlilingual bridge, we might consider how this changes, but for now, they're in English.

---

> Can you tell me an equivalent phrase/word in English then?

No, because I don't know what the word means, but I can refer you to the [first google result](https://www.urbandictionary.com/define.php?term=Oida):

<img width="637" alt="Screenshot 2020-12-21 at 16 30 45" src="https://user-images.githubusercontent.com/286476/102799060-e1b1ec00-43a9-11eb-8afa-d5610b304e66.png">